unit_tests/conftest.py

```diff
@@ -102,7 +102,7 @@ def db_alternator_up():
             logging.error("Error checking for scylla up normal: %s", details)
             return False
 
-    scylla.remoter.run('apt-get update')
+    # scylla.remoter.run('apt-get update')
```

Comment:
Is it debug left-over?

Reply:
I think this is not needed anymore, I need to put it into it's own commit

Reply:
move it to it's own commit [3cf3c4c](https://github.com/scylladb/scylla-cluster-tests/pull/7579/commits/3cf3c4cf296d6bfbb40c9ab37441b1702cf19d7a)